### ISSUE TITLE
Locize backend removes keys from json with `null` value.

### ISSUE THREAD

**Issue by Ibadichan** [locize/i18next-locize-backend#347]
## 🐛 Bug Report

Locize backend removes keys which have `null` value.

## To Reproduce
Just submit the following structure to locize:
```
{
  abc: null,
  something: [null, null]
}
```

When adding keys to locize using cli it looks like (save-missing):

![Снимок экрана 2022-06-22 в 23 14 48](https://user-images.githubusercontent.com/26125121/175127962-9549e7f1-00e8-4b7a-9309-7f705ab30b97.png)

But when i want to download locales (locize download) - it returns empty json files.

Can i configure this behavior on locize? Thank you.

**Comment by adrai** [comment]
Yes, the /update API endpoint (used by the CLI sync command) uses null to delete the keys: https://docs.locize.com/integration/api#update-remove-translations
btw: Published translation resources do ignore keys not having any value (null, empty string).
Alternatively, you can set the keys to empty strings `""` and set the preserve empty string option in your locize project's settings page.

**Comment by Ibadichan** [comment]
@adrai , thanks. But when i set "preserve empty string" option on locize it shows that content as translated. How i can configure locize to show keys with `null` or `""` as untranslated content?

**Comment by adrai** [comment]
You have to choose one or the other.
I don't understand exactly what you would like to achieve...
Can you describe your use case?

**Comment by jamuhl** [comment]
doesn't import empty string with preserveEmtpyString off work -> should add the missings and empty string is untranslated

**Comment by adrai** [comment]
Seems he want the key be present in the published translations.
But why?

**Comment by Ibadichan** [comment]
https://github.com/i18next/react-i18next/issues/1523 
I want to use i18next fallback feature with `returnObjects: true` option.

If i set `returnNull: false` option or `returnEmptyString: false` and 
set the needed key value to `null || "" ` - fallback works!

https://codesandbox.io/s/bold-snowflake-7p425r?file=/src/index.js

But when i upload locales to locize it removes `null` values and fallback does not work.
Or when i upload locales with empty strings it shows that content as translated (that is bad).

**Comment by adrai** [comment]
With the version you had yesterday (merging the 2 objects) this would have worked.

**Comment by Ibadichan** [comment]
Merging two objects is difficult when they have nested objects, arrays.
So i decided to create "placeholders" for keys that do not exist in english locale.

But locize just removes all null values or when using empty string it show the keys as translated.

**Comment by adrai** [comment]
This is a very special scenario... never seen such a use case... i18next is able to do everything, but locize has no idea how the translations are getting used in i18next...

If I understood correcty, having a locize setting to preserve null values (handling it as untranslated) would help, right?

**Comment by Ibadichan** [comment]
Yes. I need only the setting to show empty string values as untranslated content in UI.

**Comment by adrai** [comment]
We will check if we can/should introduce that...

**Comment by adrai** [comment]
Can you please send an email to support@locize.com with your project-id, this way we can try to enable that already on your project, before it get generally available?

**Comment by Ibadichan** [comment]
I sent an email with project id.

**Comment by Ibadichan** [comment]
![Снимок экрана 2022-06-23 в 16 12 36](https://user-images.githubusercontent.com/26125121/175307174-07016045-4cd3-4af5-b4b9-556111fe639b.png)
Thanks!
